src/views/components/attendees-list/attendees/attendee.php

```diff
@@ -27,9 +27,10 @@
 	return;
 }
 
-use TEC\Tickets\Commerce\Utils\Value;
-$total = Value::create( $attendee['price_paid'] );
+$provider     = Tribe__Tickets__Tickets::get_ticket_provider_instance( $attendee['provider'] );
+$ticket_price = $provider->get_price_html( $attendee['product_id'] );
+
 ?>
-<div class="tribe-tickets__commerce-order-attendees-list-attendee-details">
-	<?php echo esc_html( $attendee['ticket'] ) . ' - ' . esc_html( $total->get_currency() ); ?>
+<div class="tec-tickets__attendees-order-attendees-list-attendee-details">
+	<div><?php echo esc_html( $attendee['ticket'] ) . ' - ' . $ticket_price; ?></div>
```

Comment:
We still need to escape the ticket price, even though it's HTML, because it's filtered in some instances when it's returned:
```suggestion
	<div><?php echo esc_html( $attendee['ticket'] ) . ' - ' . wp_kses_post( $ticket_price ); ?></div>
```